pcdsdevices/mirror.py

```diff
@@ -1132,6 +1132,10 @@ class FFMirror(BaseInterface, GroupDevice, LightpathMixin):
     y_enc_rms = Cpt(PytmcSignal, ':ENC:Y:RMS', io='i', kind='normal')
     pitch_enc_rms = Cpt(PytmcSignal, ':ENC:PITCH:RMS', io='i', kind='normal')
 
+    cool_flow1 = Cpt(EpicsSignalRO, ':FWM:1_RBV', kind='normal')
```

Comment:
Is there any interesting information we could put in the doc for these Components?  Something like where these readouts are?

Reply:
yea, I can add some details here.